src/main/bash/sdkman-help.sh

```diff
@@ -25,8 +25,8 @@ function __sdk_help() {
 	__sdkman_echo_no_colour "       install   or i    <candidate> [version] [local-path]"
 	__sdkman_echo_no_colour "       uninstall or rm   <candidate> <version>"
 	__sdkman_echo_no_colour "       list      or ls   [candidate]"
-	__sdkman_echo_no_colour "       use       or u    <candidate> <version>"
-	__sdkman_echo_no_colour "       default   or d    <candidate> [version]"
+	__sdkman_echo_no_colour "       use       or u    <candidate> <identifier>"
+	__sdkman_echo_no_colour "       default   or d    <candidate> [identifier]"
```

Comment:
Version is global to SDKMAN and is true for all candidates (Java is just one of many). We have overloaded the term to accommodate Java in this case, but `Identifier` actually means `version` here.